features/login.feature

```diff
@@ -0,0 +1,29 @@
+Feature: New customer Registration
+
+
+Background: User is Logged In
+	Given the user navigates to Homepage
+	When the user clicks on the create a account button
+	Then the user should be navigated to customer account page
+
+Scenario: Empty registration Authentication (Negative scenario)
+    When the user clicks on create an account button
+    Then the user should see 5 error message
```

Comment:
@karthikeyan7010 We should stop the scenario with this check and move on to create another scenario for rest